src/App.js

```diff
@@ -30,22 +30,76 @@ const buildFields = (fields) => {
   return fields.map((field) => {
     return i18n.t(field)
   })
+}
 
+const theme = {
+  name: 'dhis2-theme',
+  tokens: {
+    components: {
+      button: {
+        link: {
+          color: {
+            value: '#0d47a1'
```

Comment:
I think it would be good to add a comment about why this theme is in place, such as "Making all of the blue match across links and tabs"